netbout-web/src/main/java/com/netbout/rest/MarkdownPegdown.java

```diff
@@ -156,7 +158,12 @@ private static Tidy makeTidy() {
         final Tidy tidy = new Tidy();
         tidy.setShowErrors(0);
         tidy.setErrout(
-            new PrintWriter(Logger.stream(Level.FINE, MarkdownPegdown.class))
+            new PrintWriter(
```

Comment:
@exper0 btw what exactly Findbugs says about it?
PrintWriter has OutputStream in its constructor and Logger.stream returns OutputStream too